README.md

```diff
@@ -125,6 +126,22 @@ MONGOMS_MD5_CHECK=1 # if you want to make MD5 check of downloaded binary.
 # Passed constructor parameter `binary.checkMD5` has higher priority.
 ```
 
+```json
+"config": {
+  "mongoDbMemoryServer": {
```

Comment:
According to the naming of package `mongodb-memory-server` better will be to use the following key `mongodbMemoryServer` (lowercased D).